transport/tchannel/outbound_test.go

```diff
@@ -124,9 +124,10 @@ func TestCallSuccess(t *testing.T) {
 		headerVal = "FooBarBaz"
 	)
 	tests := []struct {
-		msg       string
-		options   []TransportOption
-		headerVal []byte
+		msg        string
+		options    []TransportOption
+		headerVal  []byte
+		addSvcName bool
```

Comment:
nit: maybe `withServiceResponseHeader`/`addServiceResponseHeader`/ `withServiceRespHeader` / or something similar. Just to indicate that this is a header we're adding.

Reply:
Modified